Zend/tests/function_arguments/sensitive_parameter_correctly_captures_original.phpt

```diff
@@ -0,0 +1,70 @@
+--TEST--
+The SensitiveParameterValue replacement value correctly captures the original value.
+--FILE--
+<?php
+ 
+function test(
+    $foo,
+    #[\SensitiveParameter] $bar,
+    $baz
+) {
+    throw new \Exception('Error');
+}
+ 
+try {
+    test('foo', 'bar', 'baz');
+    echo 'Not reached';
+} catch (\Exception $e) {
+    echo $e->getMessage(), PHP_EOL;
+    $testFrame = $e->getTrace()[0];
+    \var_dump($testFrame['function']);
+    \var_dump(\count($testFrame['args']));
+    \var_dump($testFrame['args'][0]);
+    \assert($testFrame['args'][1] instanceof \SensitiveParameterValue);
```

Comment:
I'd go with a `var_dump()`; alternatively, assertions could be enabled in an `--INI--` section.

Reply:
I've added theses `assert()`s primarily as IDE assistance, so that it knows that a `->getValue()` methods exists (and what it shall do). Even if assertions are disabled, the test will fail in the next line if the assertion would be violated. 

A `var_dump()` doesn't solve the original intent, but I certainly don't want to do something unusual. Would you recommend:

1. Add
```
--INI--
zend.assertions=1
assert.active=1
```
2. Remove the line completely.
3. Leave it as-is.

?